ocaml/xapi/xapi_vgpu.ml

```diff
@@ -42,19 +62,27 @@ let create' ~__context  ~vM ~gPU_group ~device ~other_config ~_type ~powerstate_
                     (Api_errors.invalid_value, ["type"; Ref.string_of _type]))
     end
   in
+  (* during multiple vgpus creation:
+     1. Underlying vgpu_type should support multiple
+     2. _type must be listed on the all vgpu_type's compatible lists*)
+  let existing = Db.VM.get_VGPUs ~__context ~self:vM in
+  if existing <> [] then begin
```

Comment:
You don't really need this if-statement.

Reply:
fixed in new commit